OpenSim/Auxiliary/auxiliaryTestFunctions.h

```diff
@@ -79,16 +79,16 @@ inline void ASSERT(bool cond,
  */
 void CHECK_STORAGE_AGAINST_STANDARD(OpenSim::Storage& result, 
                                     OpenSim::Storage& standard, 
-                                    OpenSim::Array<double> tolerances, 
+                                    std::vector<double> tolerances, 
                                     std::string testFile, 
                                     int testFileLine, 
                                     std::string errorMessage)
 {
```

Comment:
I'm not sure if this should be templatized. In the end, `tolerances` is compared to `comparisons`, which is necessarily a vector of `double` because of [this line](https://github.com/opensim-org/opensim-core/blob/4c416d4d000e4cfcbbe59b64f40c6f4140359076/OpenSim/Common/Storage.cpp#L3338). I could easily be missing something though?